static/styles/hotspots.scss

```diff
@@ -26,12 +26,13 @@
         background-color: hsl(0, 0%, 100%);
         border-radius: 50%;
         border: 1px solid hsl(205, 100%, 70%);
-        animation: pulsate 1.5s ease-out;
-        animation-iteration-count: infinite;
+        transform: scale(2.2, 2.2);
```

Comment:
Can this be shortened to `transform: scale(2.2);` similar to other places you changed?

Reply:
Shortened.